core/drivers/stm32_uart.c

```diff
@@ -149,7 +152,19 @@ struct stm32_uart_pdata *stm32_uart_init_from_dt_node(void *fdt, int node)
 	pd->chip.ops = &stm32_uart_serial_ops;
 	pd->base.pa = info.reg;
 	pd->secure = (info.status == DT_STATUS_OK_SEC);
-	pd->clock = (unsigned int)info.clock;
+
+	if (fdt_getprop(fdt, node, "clocks", NULL)) {
+		res = clk_dt_get_by_index(fdt, node, 0, &pd->clock);
+		if (!res) {
+			assert(pd->clock);
+
+			res = clk_enable(pd->clock);
+			if (res)
+				panic();
+		}
+	}
+	if (!pd->clock)
```

Comment:
missing empty line before

Reply:
I'll discard this change since your comment above.